content/intro-to-writing-data.md

```diff
@@ -66,78 +85,15 @@ The `sendAndConfirmTransaction()` functions takes as parameters
 - a transaction
 - an array of keypairs that will act as signers on the transaction - in this example, we only have the one signer: the sender.
 
-### Instructions
```

Comment:
So this has been moved to the next chapter, after students have practiced transferring SOL